docs/AboutUs.md

```diff
@@ -3,57 +3,43 @@ layout: page
 title: About Us
 ---
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
+## Meet our team!
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+**SociaLite** is a project is built for CS2103T Software Engineering.
```

Comment:
Remove the word 'is' before "built for CS2103T Software Engineering."

Reply:
Thanks for the catch! Will remove and commit later on.